src/run.test.ts

```diff
@@ -205,7 +205,4 @@ describe('Set context', () => {
       expect(fs.chmodSync).toBeCalledWith(kubeconfigPath, '600')
       expect(core.exportVariable).toBeCalledWith('KUBECONFIG', kubeconfigPath)
    })
-
```

Comment:
nit: newlines

Reply:
that got added by the prettier formatter i can try to remove it

Reply:
yeah those are out of our format spec we can't add them back without an override

Reply:
ah ok nw nw